istioctl/cmd/istioctl/main.go

```diff
@@ -56,6 +57,9 @@ import (
 
 const (
 	kubePlatform = "kube"
+
+	// Headings for short format listing of unknown types
+	unknownShortOutputHeading = "NAME\tKIND\tNAMESPACE"
```

Comment:
here and elsewhere... is it safe to use tabs for alignment? isn't it risky if two cells in the same tab separated column have content lengths that are very different from each other? does `istioctl` generally use tabs for alignment?

Reply:
Until May there was a problem with _istioctl_ alignment.  It was fixed by https://github.com/istio/istio/commit/71ade65b88b775d9e85ded5049e0f827bbdfa8d9#diff-686b5aacecbdcbce4a7ea3ac84d4b658 .  I was surprised when I saw the fix; it turns out _tabwriter_ is smarter than first appears; it buffers internally until Flush().  See https://golang.org/pkg/text/tabwriter/